http/http_server.hpp

```diff
@@ -163,6 +169,16 @@ class Server
                     }
                     this->startAsyncWaitForSignal();
                 }
+#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
+                if (signalNo == SIGUSR1)
```

Comment:
We can't kill bmcweb to get this data

Reply:
SIGUSR1 is a user signal. This wouldn't kill any process. Its commonly used by linux applications to override and implement specific actions.

Reply:
As mentioned in today's SoS, the Linux `kill` command sends a signal to the process (https://en.wikipedia.org/wiki/Signal_(IPC)).  It does not necessarily send the terminate or kill signal.  And it is common to make long-running daemons respond to various signals